parkings/tests/api/operator/test_disc_parking.py

```diff
@@ -34,6 +35,7 @@ def disc_parking_data():
 
 
 def test_post_disc_parking(operator_api_client, operator, disc_parking_data):
+    create_enforced_domain_for_user(operator.user)
```

Comment:
I don't understand why operator would have to be an enforcer here. Are you sure this enforcer-creation is needed?